FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -18,9 +20,31 @@
 from DIRAC.Core.Security.X509Chain import X509Chain
 import DIRAC
 from DIRAC.Core.Utilities import InstallTools
+from DIRAC.FrameworkSystem.Client.ComponentMonitoringClient import ComponentMonitoringClient
+from DIRAC.Core.Utilities.ThreadScheduler import gThreadScheduler
+from DIRAC.FrameworkSystem.Client.SystemAdministratorClient import SystemAdministratorClient
 
 class SystemAdministratorHandler( RequestHandler ):
 
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    """
+    Handler class initialization
+    """
+
+    hostMonitoring = True
+
+    # Check the flag for monitoring of the state of the host
+    hostMonitoring = gConfig.getValue( 'DIRAC/HostMonitoring', 'True' )
```

Comment:
As a location: it should be dependent from the setup. I suggest you do:

hostMonitoring = cls.srv_getCSOption('HostMonitoring', True)